### ISSUE TITLE
OWS SOCKS5 proxy reverts to "NO PROXY"

### ISSUE THREAD

**Issue by IanWardell** [karakun/OpenWebStart#360]
**OS:** Windows 10 
**JVM Manager:** 11.0.7 Azul System, Inc. Windows x64

**Issue:** When loading a iDRAC6 Virtual Console Client .jnlp file I cannot get it to run over a socks5 proxy at localhost:8080 
- Proxy verified working via firefox and ssh
- Tested on both manual proxy (set to only socks5) and to Firefox proxy 

.jnlp file runs fine when I am connected to the network via VPN. 

**Recreate:** 
1. Fresh install OWS
2. use cmd to create a ssh socks5 proxy 
3. Set SOCKS5 Proxy in OWS
4. Run .jnlp file

**Verbose logging output**

### Console 1
```
About to launch external with commands: '[C:\Users\good4y0u\.cache\icedtea-web\jvm-cache\azul_11.0.7\bin\java.exe, -Xbootclasspath/a:C:\Users\good4y0u\AppData\Local\Programs\OpenWebStart\openwebstart.jar, --add-exports=java.base/sun.net.www.protocol.jar=ALL-UNNAMED,java.desktop, --add-exports=java.base/jdk.internal.util.jar=ALL-UNNAMED,java.desktop, --add-exports=java.base/com.sun.net.ssl.internal.ssl=ALL-UNNAMED,java.desktop, --add-reads=java.naming=ALL-UNNAMED,java.desktop, --add-exports=java.desktop/sun.awt.X11=ALL-UNNAMED,java.desktop, --add-exports=java.desktop/sun.applet=ALL-UNNAMED,java.desktop,jdk.jsobject, --add-exports=java.base/sun.security.action=ALL-UNNAMED,java.desktop, --add-reads=java.base=ALL-UNNAMED,java.desktop, --add-exports=java.base/sun.net.www.protocol.http=ALL-UNNAMED,java.desktop, --add-exports=java.naming/com.sun.jndi.toolkit.url=ALL-UNNAMED,java.desktop, --add-exports=java.base/sun.security.util=ALL-UNNAMED,java.desktop, --add-reads=java.desktop=ALL-UNNAMED,java.naming, --add-exports=java.desktop/sun.awt=ALL-UNNAMED,java.desktop, --add-exports=java.base/sun.security.x509=ALL-UNNAMED,java.desktop, --add-exports=java.desktop/javax.jnlp=ALL-UNNAMED,java.desktop, --add-exports=java.base/sun.security.provider=ALL-UNNAMED,java.desktop, --add-exports=java.base/sun.security.validator=ALL-UNNAMED,java.desktop, net.sourceforge.jnlp.runtime.Boot, -Xnofork, C:\jnlp\viewer.jnlp]'
Saving runtime cache to filesystem
Replacing runtime definition with new one
using java runtime at 'C:\Users\good4y0u\.cache\icedtea-web\jvm-cache\azul_11.0.7' for launching managed application
Found JVM JavaRuntime{version=11.0.7, vendor=Azul Systems, Inc., operationSystem=WIN64}
No newer version was installed
Received 18 possible runtime definitions from server
No proxy found for 'socket://download-openwebstart.com:443'. Falling back to NO_PROXY
No proxy found for 'https://download-openwebstart.com/jvms.json'. Falling back to NO_PROXY
Endpoint to request for Java runtimes: https://download-openwebstart.com/jvms.json
Trying to find remote Java runtime. Requested version: '1.6+' Requested vendor: '*' requested os: 'WIN64'
Local runtime JavaRuntime{version=11.0.7, vendor=Azul Systems, Inc., operationSystem=WIN64} found but remote endpoint is checked for newer versions
Trying to find local Java runtime. Requested version: '1.6+' Requested vendor: '*' requested os: 'WIN64' active: 'true'
Trying to find Java runtime. Requested version: '1.6+', vendor: '*', os: 'WIN64', server-url: 'null'
searching for JRE with version string '1.6+' and vendor '*'
Saving runtime cache to filesystem
Adding runtime definition
Saving runtime cache to filesystem
Adding runtime definition
Saving runtime cache to filesystem
Adding runtime definition
Saving runtime cache to filesystem
Adding runtime definition
Saving runtime cache to filesystem
Clearing runtime cache
Loading runtime cache from filesystem
result: null
arg: net.sourceforge.jnlp.JNLPFile@58e40182
call privileged method: checkSingleInstanceRunning
Jars not ready to provide attribute Application-Name
Jars not ready to provide attribute Application-Name
Acquired shared lock on C:\Users\good4y0u~1\AppData\Local\Temp\good4y0u\netx\locks\netx_running to indicate javaws is running
good - your JRE - 1.8.0_265 - match requested JRE - 1.6+
Acceptable vendor tag found, contains: Dell Inc.
Jars not ready to provide attribute Application-Name
Description: null
Homepage: null
Using MalformedXMLParser
Will add resource 'viewer.jnlp'
Will add resource at location 'file:/C:/jnlp/viewer.jnlp'
good - your JRE - 1.8.0_265 - match requested JRE - 1.6+
Acceptable vendor tag found, contains: Dell Inc.
Jars not ready to provide attribute Application-Name
Description: null
Homepage: null
Using MalformedXMLParser
Download done. Shutting down executor
Resource is not cacheable: viewer.jnlp
Checking download state of viewer.jnlp
Download for viewer.jnlp has not been started until now
Will check and maybe put into cache: viewer.jnlp
Will add resource 'viewer.jnlp'
Will add resource at location 'file:/C:/jnlp/viewer.jnlp'
JNLP file location: C:\jnlp\viewer.jnlp
Proceeding with jnlp
proxy bypass list: []
Socks proxy congfig: host localhost - port 8080
Ftp proxy congfig: host null - port -1
Https proxy congfig: host null - port -1
Http proxy congfig: host null - port -1
Manual proxy created
Selected ProxyProvider : MANUAL_HOSTS
Operating Keystore C:\Users\good4y0u\.config\icedtea-web\security\trusted.clientcerts
Loading Keystore C:\Users\good4y0u\.config\icedtea-web\security\trusted.clientcerts
Keystore file C:\Users\good4y0u\.config\icedtea-web\security\trusted.clientcerts exists.
Operating Keystore Unknown
Loading Keystore Unknown
Keystore file c:\users\good4y0u\appdata\local\programs\openwebstart\jre\lib\security\trusted.clientcerts does not exists.
Operating Keystore C:\Users\good4y0u\.config\icedtea-web\security\trusted.cacerts
Loading Keystore C:\Users\good4y0u\.config\icedtea-web\security\trusted.cacerts
Keystore file C:\Users\good4y0u\.config\icedtea-web\security\trusted.cacerts exists.
Operating Keystore C:\Users\good4y0u\.config\icedtea-web\security\trusted.jssecacerts
Loading Keystore C:\Users\good4y0u\.config\icedtea-web\security\trusted.jssecacerts
Keystore file C:\Users\good4y0u\.config\icedtea-web\security\trusted.jssecacerts exists.
Operating Keystore c:\users\good4y0u\appdata\local\programs\openwebstart\jre\lib\security\cacerts
Loading Keystore c:\users\good4y0u\appdata\local\programs\openwebstart\jre\lib\security\cacerts
Keystore file c:\users\good4y0u\appdata\local\programs\openwebstart\jre\lib\security\cacerts exists.
Operating Keystore Unknown
Loading Keystore Unknown
Keystore file c:\users\good4y0u\appdata\local\programs\openwebstart\jre\lib\security\jssecacerts does not exists.
Operating Keystore C:\Users\good4y0u\.config\icedtea-web\security\trusted.certs
Loading Keystore C:\Users\good4y0u\.config\icedtea-web\security\trusted.certs
Keystore file C:\Users\good4y0u\.config\icedtea-web\security\trusted.certs exists.
Operating Keystore C:\Users\good4y0u\.config\icedtea-web\security\trusted.jssecerts
Loading Keystore C:\Users\good4y0u\.config\icedtea-web\security\trusted.jssecerts
Keystore file C:\Users\good4y0u\.config\icedtea-web\security\trusted.jssecerts exists.
Operating Keystore Unknown
Loading Keystore Unknown
Keystore file c:\users\good4y0u\appdata\local\programs\openwebstart\jre\lib\security\trusted.certs does not exists.
Operating Keystore Unknown
Loading Keystore Unknown
Keystore file c:\users\good4y0u\appdata\local\programs\openwebstart\jre\lib\security\trusted.jssecerts does not exists.
Operating Keystore Unknown
Operating Keystore C:\Users\good4y0u\.config\icedtea-web\security\trusted.clientcerts
Loading Keystore C:\Users\good4y0u\.config\icedtea-web\security\trusted.clientcerts
Keystore file C:\Users\good4y0u\.config\icedtea-web\security\trusted.clientcerts exists.
Starting security dialog thread
using com.openwebstart.extensionpoint.OwsExtensionPoint extension point
Java Runtime AdoptOpenJDK-1.8.0_265
OS: Windows 10
OpenWebStartLauncher called with args: [C:\jnlp\viewer.jnlp].
Calling ITW Boot with args [C:\jnlp\viewer.jnlp].
RelevantJavawsArgs: '[C:\jnlp\viewer.jnlp]'
Checking if installation time (1606169422449) is after last initial config time (1606169430157)
Setting download indicator to com.openwebstart.download.ApplicationDownloadIndicator@27808f31
Trying to set download indicator
Property 'ows.install4j.propertyUpdate' is unknown.
Property 'deployment.proxy' is unknown.
Loading USER level properties from: file:/C:/Users/good4y0u/.config/icedtea-web/deployment.properties
Property 'ows.install4j.propertyUpdate' is unknown.
Property 'deployment.proxy' is unknown.
Loading USER level properties from: file:/C:/Users/good4y0u/.config/icedtea-web/deployment.properties
Ico provider registered correctly.
Ico provider registered correctly.
Starting OpenWebStart 1.3.0
OWS main args [C:\jnlp\viewer.jnlp].
init logger factory to net.sourceforge.jnlp.util.logging.OutputControllerLoggerFactory@694f9431
```

### Console 2
```
Waiting for exception dialog to be closed
Launch exception
netx: Initialization Error: Could not initialize application. (Fatal: Initialization Error: Unknown Main-Class. Could not determine the main class for this application.)
Error flag set for resource 'https://192.168.1.99:443/software/avctVMWin64.jar'. Can not return a local file for the resource
Activate native: https://192.168.1.99:443/software/avctVMWin64.jar
Activate jar: https://192.168.1.99:443/software/avctVMWin64.jar
Error flag set for resource 'https://192.168.1.99:443/software/avctVMWin64.jar'. Can not return a local file for the resource
DownloadServiceListener will be disposed
Download done. Shutting down executor
Download for avctVMWin64.jar has already been started.
Will check and maybe put into cache: avctVMWin64.jar
DownloadServiceListener for iDRAC6 Virtual Console Client will be created
Jars not ready to provide attribute Application-Name
checkForMain JAR https://192.168.1.99:443/software/avctVMWin64.jar not found. Continuing.
Error flag set for resource 'https://192.168.1.99:443/software/avctVMWin64.jar'. Can not return a local file for the resource
Error flag set for resource 'https://192.168.1.99:443/software/avctKVMIOWin64.jar'. Can not return a local file for the resource
Activate native: https://192.168.1.99:443/software/avctKVMIOWin64.jar
Activate jar: https://192.168.1.99:443/software/avctKVMIOWin64.jar
Error flag set for resource 'https://192.168.1.99:443/software/avctKVMIOWin64.jar'. Can not return a local file for the resource
DownloadServiceListener will be disposed
Download done. Shutting down executor
Download for avctKVMIOWin64.jar has already been started.
Will check and maybe put into cache: avctKVMIOWin64.jar
DownloadServiceListener for iDRAC6 Virtual Console Client will be created
Jars not ready to provide attribute Application-Name
checkForMain JAR https://192.168.1.99:443/software/avctKVMIOWin64.jar not found. Continuing.
Error flag set for resource 'https://192.168.1.99:443/software/avctKVMIOWin64.jar'. Can not return a local file for the resource
Error flag set for resource 'https://192.168.1.99:443/software/avctKVM.jar'. Can not return a local file for the resource
Activate native: https://192.168.1.99:443/software/avctKVM.jar
Activate jar: https://192.168.1.99:443/software/avctKVM.jar
Error flag set for resource 'https://192.168.1.99:443/software/avctKVM.jar'. Can not return a local file for the resource
DownloadServiceListener will be disposed
Download done. Shutting down executor
Download for avctKVM.jar has already been started.
Will check and maybe put into cache: avctKVM.jar
DownloadServiceListener for iDRAC6 Virtual Console Client will be created
Jars not ready to provide attribute Application-Name
checkForMain JAR https://192.168.1.99:443/software/avctKVM.jar not found. Continuing.
Error flag set for resource 'https://192.168.1.99:443/software/avctKVM.jar'. Can not return a local file for the resource
checkForMain JAR https://192.168.1.99:443/software/avctVMWin64.jar not found. Continuing.
Error flag set for resource 'https://192.168.1.99:443/software/avctVMWin64.jar'. Can not return a local file for the resource
checkForMain JAR https://192.168.1.99:443/software/avctKVMIOWin64.jar not found. Continuing.
Error flag set for resource 'https://192.168.1.99:443/software/avctKVMIOWin64.jar'. Can not return a local file for the resource
checkForMain JAR https://192.168.1.99:443/software/avctKVM.jar not found. Continuing.
Error flag set for resource 'https://192.168.1.99:443/software/avctKVM.jar'. Can not return a local file for the resource
Error flag set for resource 'https://192.168.1.99:443/software/avctVMWin64.jar'. Can not return a local file for the resource
Error flag set for resource 'https://192.168.1.99:443/software/avctKVMIOWin64.jar'. Can not return a local file for the resource
Error flag set for resource 'https://192.168.1.99:443/software/avctKVM.jar'. Can not return a local file for the resource
DownloadServiceListener will be disposed
Download done. Shutting down executor
could not download resource location=https://192.168.1.99:443/software/avctVMWin64.jar version=null state=INCOMPLETE from any of theses urls [https://192.168.1.99:443/software/avctVMWin64.jar]
Exception while downloading resource location=https://192.168.1.99:443/software/avctVMWin64.jar version=null state=INCOMPLETE from https://192.168.1.99:443/software/avctVMWin64.jar - Connection timed out: connect
could not download resource location=https://192.168.1.99:443/software/avctKVM.jar version=null state=INCOMPLETE from any of theses urls [https://192.168.1.99:443/software/avctKVM.jar]
could not download resource location=https://192.168.1.99:443/software/avctKVMIOWin64.jar version=null state=INCOMPLETE from any of theses urls [https://192.168.1.99:443/software/avctKVMIOWin64.jar]
Exception while downloading resource location=https://192.168.1.99:443/software/avctKVM.jar version=null state=INCOMPLETE from https://192.168.1.99:443/software/avctKVM.jar - Connection timed out: connect
Exception while downloading resource location=https://192.168.1.99:443/software/avctKVMIOWin64.jar version=null state=INCOMPLETE from https://192.168.1.99:443/software/avctKVMIOWin64.jar - Connection timed out: connect
No proxy found for 'socket://192.168.1.99:443'. Falling back to NO_PROXY
No proxy found for 'https://192.168.1.99:443/software/avctVMWin64.jar'. Falling back to NO_PROXY
Will download in background: https://192.168.1.99:443/software/avctVMWin64.jar
Failed to determine best URL for location=https://192.168.1.99:443/software/avctVMWin64.jar version=null state=INCOMPLETE will try all of [https://192.168.1.99:443/software/avctVMWin64.jar]
failed to determine best URL: java.util.concurrent.ExecutionException: java.net.SocketTimeoutException: connect timed out
While processing https://192.168.1.99:443/software/avctVMWin64.jar  by HEAD for resource location=https://192.168.1.99:443/software/avctVMWin64.jar version=null state=INCOMPLETE got connect timed out
URL connection 'https://192.168.1.99:443/software/avctVMWin64.jar' header fields: {}
Following exception should be harmless, but may help in finding root cause.
No proxy found for 'socket://192.168.1.99:443'. Falling back to NO_PROXY
No proxy found for 'socket://192.168.1.99:443'. Falling back to NO_PROXY
No proxy found for 'https://192.168.1.99:443/software/avctKVM.jar'. Falling back to NO_PROXY
No proxy found for 'https://192.168.1.99:443/software/avctKVMIOWin64.jar'. Falling back to NO_PROXY
Will download in background: https://192.168.1.99:443/software/avctKVMIOWin64.jar
Failed to determine best URL for location=https://192.168.1.99:443/software/avctKVMIOWin64.jar version=null state=INCOMPLETE will try all of [https://192.168.1.99:443/software/avctKVMIOWin64.jar]
Will download in background: https://192.168.1.99:443/software/avctKVM.jar
Failed to determine best URL for location=https://192.168.1.99:443/software/avctKVM.jar version=null state=INCOMPLETE will try all of [https://192.168.1.99:443/software/avctKVM.jar]
failed to determine best URL: java.util.concurrent.ExecutionException: java.net.SocketTimeoutException: connect timed out
While processing https://192.168.1.99:443/software/avctKVMIOWin64.jar  by HEAD for resource location=https://192.168.1.99:443/software/avctKVMIOWin64.jar version=null state=INCOMPLETE got connect timed out
failed to determine best URL: java.util.concurrent.ExecutionException: java.net.SocketTimeoutException: connect timed out
URL connection 'https://192.168.1.99:443/software/avctKVMIOWin64.jar' header fields: {}
While processing https://192.168.1.99:443/software/avctKVM.jar  by HEAD for resource location=https://192.168.1.99:443/software/avctKVM.jar version=null state=INCOMPLETE got connect timed out
Following exception should be harmless, but may help in finding root cause.
URL connection 'https://192.168.1.99:443/software/avctKVM.jar' header fields: {}
Following exception should be harmless, but may help in finding root cause.
Download for avctVMWin64.jar has already been started.
Will check and maybe put into cache: avctVMWin64.jar
Download for avctKVMIOWin64.jar has already been started.
Will check and maybe put into cache: avctKVMIOWin64.jar
Download for avctKVM.jar has already been started.
Will check and maybe put into cache: avctKVM.jar
No proxy found for 'socket://192.168.1.99:443'. Falling back to NO_PROXY
No proxy found for 'https://192.168.1.99:443/software/avctVMWin64.jar'. Falling back to NO_PROXY
Candidate URLs for location=https://192.168.1.99:443/software/avctVMWin64.jar version=null state=INCOMPLETE: [https://192.168.1.99:443/software/avctVMWin64.jar]
needsUpdateCheck: https://192.168.1.99:443/software/avctVMWin64.jar -> true
No proxy found for 'socket://192.168.1.99:443'. Falling back to NO_PROXY
No proxy found for 'https://192.168.1.99:443/software/avctKVMIOWin64.jar'. Falling back to NO_PROXY
isCached: https://192.168.1.99:443/software/avctVMWin64.jar - (v: null) = false
DownloadServiceListener for iDRAC6 Virtual Console Client will be created
Candidate URLs for location=https://192.168.1.99:443/software/avctKVMIOWin64.jar version=null state=INCOMPLETE: [https://192.168.1.99:443/software/avctKVMIOWin64.jar]
Jars not ready to provide attribute Application-Name
Download of resource avctVMWin64.jar will start now!
Download has not been started yet: avctVMWin64.jar
needsUpdateCheck: https://192.168.1.99:443/software/avctKVMIOWin64.jar -> true
No proxy found for 'socket://192.168.1.99:443'. Falling back to NO_PROXY
Checking download state of avctVMWin64.jar
No proxy found for 'https://192.168.1.99:443/software/avctKVM.jar'. Falling back to NO_PROXY
isCached: https://192.168.1.99:443/software/avctKVMIOWin64.jar - (v: null) = false
Download for avctVMWin64.jar has not been started until now
Will check and maybe put into cache: avctVMWin64.jar
Download of resource avctKVMIOWin64.jar will start now!
Candidate URLs for location=https://192.168.1.99:443/software/avctKVM.jar version=null state=INCOMPLETE: [https://192.168.1.99:443/software/avctKVM.jar]
Will add resource 'avctVMWin64.jar'
needsUpdateCheck: https://192.168.1.99:443/software/avctKVM.jar -> true
Will add resource at location 'https://192.168.1.99:443/software/avctVMWin64.jar'
Download has not been started yet: avctKVMIOWin64.jar
Checking download state of avctKVMIOWin64.jar
isCached: https://192.168.1.99:443/software/avctKVM.jar - (v: null) = false
Download for avctKVMIOWin64.jar has not been started until now
Will check and maybe put into cache: avctKVMIOWin64.jar
Will add resource 'avctKVMIOWin64.jar'
Download of resource avctKVM.jar will start now!
Will add resource at location 'https://192.168.1.99:443/software/avctKVMIOWin64.jar'
Download has not been started yet: avctKVM.jar
Checking download state of avctKVM.jar
Download for avctKVM.jar has not been started until now
Will check and maybe put into cache: avctKVM.jar
Will add resource 'avctKVM.jar'
Will add resource at location 'https://192.168.1.99:443/software/avctKVM.jar'
New classloader: file:/C:/jnlp/viewer.jnlp
Error loading splash image: https://192.168.1.99:443/images/logo.gif
Error flag set for resource 'https://192.168.1.99:443/images/logo.gif'. Can not return a local file for the resource
Download done. Shutting down executor
could not download resource location=https://192.168.1.99:443/images/logo.gif version=null state=INCOMPLETE from any of theses urls [https://192.168.1.99:443/images/logo.gif]
Exception while downloading resource location=https://192.168.1.99:443/images/logo.gif version=null state=INCOMPLETE from https://192.168.1.99:443/images/logo.gif - Connection timed out: connect
No proxy found for 'socket://192.168.1.99:443'. Falling back to NO_PROXY
No proxy found for 'https://192.168.1.99:443/images/logo.gif'. Falling back to NO_PROXY
Will download in background: https://192.168.1.99:443/images/logo.gif
Failed to determine best URL for location=https://192.168.1.99:443/images/logo.gif version=null state=INCOMPLETE will try all of [https://192.168.1.99:443/images/logo.gif]
failed to determine best URL: java.util.concurrent.ExecutionException: java.net.SocketTimeoutException: connect timed out
While processing https://192.168.1.99:443/images/logo.gif  by HEAD for resource location=https://192.168.1.99:443/images/logo.gif version=null state=INCOMPLETE got connect timed out
URL connection 'https://192.168.1.99:443/images/logo.gif' header fields: {}
Following exception should be harmless, but may help in finding root cause.
No proxy found for 'socket://192.168.1.99:443'. Falling back to NO_PROXY
Download for logo.gif has already been started.
Will check and maybe put into cache: logo.gif
No proxy found for 'https://192.168.1.99:443/images/logo.gif'. Falling back to NO_PROXY
Candidate URLs for location=https://192.168.1.99:443/images/logo.gif version=null state=INCOMPLETE: [https://192.168.1.99:443/images/logo.gif]
needsUpdateCheck: https://192.168.1.99:443/images/logo.gif -> true
isCached: https://192.168.1.99:443/images/logo.gif - (v: null) = false
Download of resource logo.gif will start now!
Download has not been started yet: logo.gif
Checking download state of logo.gif
Download for logo.gif has not been started until now
Will check and maybe put into cache: logo.gif
Will add resource 'logo.gif'
Will add resource at location 'https://192.168.1.99:443/images/logo.gif'
result: null
arg: net.sourceforge.jnlp.JNLPFile@31fbebe7
call privileged method: checkSingleInstanceRunning
Jars not ready to provide attribute Application-Name
Jars not ready to provide attribute Application-Name
Acquired shared lock on C:\Users\good4y0u~1\AppData\Local\Temp\good4y0u\netx\locks\netx_running to indicate javaws is running
good - your JRE - 11.0.7 - match requested JRE - 1.6+
Acceptable vendor tag found, contains: Dell Inc.
Jars not ready to provide attribute Application-Name
Description: null
Homepage: null
Using MalformedXMLParser
Download done. Shutting down executor
Resource is not cacheable: viewer.jnlp
Checking download state of viewer.jnlp
Download for viewer.jnlp has not been started until now
Will check and maybe put into cache: viewer.jnlp
Will add resource 'viewer.jnlp'
Will add resource at location 'file:/C:/jnlp/viewer.jnlp'
good - your JRE - 11.0.7 - match requested JRE - 1.6+
Acceptable vendor tag found, contains: Dell Inc.
Jars not ready to provide attribute Application-Name
Description: null
Homepage: null
Using MalformedXMLParser
Download done. Shutting down executor
Resource is not cacheable: viewer.jnlp
Checking download state of viewer.jnlp
Download for viewer.jnlp has not been started until now
Will check and maybe put into cache: viewer.jnlp
Will add resource 'viewer.jnlp'
Will add resource at location 'file:/C:/jnlp/viewer.jnlp'
JNLP file location: C:\jnlp\viewer.jnlp
Proceeding with jnlp
proxy bypass list: []
Socks proxy congfig: host localhost - port 8080
Ftp proxy congfig: host null - port -1
Https proxy congfig: host null - port -1
Http proxy congfig: host null - port -1
Manual proxy created
Selected ProxyProvider : MANUAL_HOSTS
Operating Keystore C:\Users\good4y0u\.config\icedtea-web\security\trusted.clientcerts
Loading Keystore C:\Users\good4y0u\.config\icedtea-web\security\trusted.clientcerts
Keystore file C:\Users\good4y0u\.config\icedtea-web\security\trusted.clientcerts exists.
Operating Keystore Unknown
Loading Keystore Unknown
Keystore file C:\Users\good4y0u\.cache\icedtea-web\jvm-cache\azul_11.0.7\lib\security\trusted.clientcerts does not exists.
Operating Keystore C:\Users\good4y0u\.config\icedtea-web\security\trusted.cacerts
Loading Keystore C:\Users\good4y0u\.config\icedtea-web\security\trusted.cacerts
Keystore file C:\Users\good4y0u\.config\icedtea-web\security\trusted.cacerts exists.
Operating Keystore C:\Users\good4y0u\.config\icedtea-web\security\trusted.jssecacerts
Loading Keystore C:\Users\good4y0u\.config\icedtea-web\security\trusted.jssecacerts
Keystore file C:\Users\good4y0u\.config\icedtea-web\security\trusted.jssecacerts exists.
Operating Keystore C:\Users\good4y0u\.cache\icedtea-web\jvm-cache\azul_11.0.7\lib\security\cacerts
Loading Keystore C:\Users\good4y0u\.cache\icedtea-web\jvm-cache\azul_11.0.7\lib\security\cacerts
Keystore file C:\Users\good4y0u\.cache\icedtea-web\jvm-cache\azul_11.0.7\lib\security\cacerts exists.
Operating Keystore Unknown
Loading Keystore Unknown
Keystore file C:\Users\good4y0u\.cache\icedtea-web\jvm-cache\azul_11.0.7\lib\security\jssecacerts does not exists.
Operating Keystore C:\Users\good4y0u\.config\icedtea-web\security\trusted.certs
Loading Keystore C:\Users\good4y0u\.config\icedtea-web\security\trusted.certs
Keystore file C:\Users\good4y0u\.config\icedtea-web\security\trusted.certs exists.
Operating Keystore C:\Users\good4y0u\.config\icedtea-web\security\trusted.jssecerts
Loading Keystore C:\Users\good4y0u\.config\icedtea-web\security\trusted.jssecerts
Keystore file C:\Users\good4y0u\.config\icedtea-web\security\trusted.jssecerts exists.
Operating Keystore Unknown
Loading Keystore Unknown
Keystore file C:\Users\good4y0u\.cache\icedtea-web\jvm-cache\azul_11.0.7\lib\security\trusted.certs does not exists.
Operating Keystore Unknown
Loading Keystore Unknown
Keystore file C:\Users\good4y0u\.cache\icedtea-web\jvm-cache\azul_11.0.7\lib\security\trusted.jssecerts does not exists.
Operating Keystore Unknown
Operating Keystore C:\Users\good4y0u\.config\icedtea-web\security\trusted.clientcerts
Loading Keystore C:\Users\good4y0u\.config\icedtea-web\security\trusted.clientcerts
Keystore file C:\Users\good4y0u\.config\icedtea-web\security\trusted.clientcerts exists.
Starting security dialog thread
Property 'ows.install4j.propertyUpdate' is unknown.
Property 'deployment.proxy' is unknown.
Loading USER level properties from: file:/C:/Users/good4y0u/.config/icedtea-web/deployment.properties
Ico provider registered correctly.
using com.openwebstart.extensionpoint.OwsExtensionPoint extension point
Java Runtime Azul Systems, Inc.-11.0.7
OS: Windows 10
OpenWebStartLauncher called with args: [-Xnofork, C:\jnlp\viewer.jnlp].
init logger factory to net.sourceforge.jnlp.util.logging.OutputControllerLoggerFactory@64b8f8f4
```

### Final Error 
```
net.sourceforge.jnlp.LaunchException: Fatal: Initialization Error: Could not initialize application. The application has not been initialized, for more information execute javaws from the command line.
	at net.sourceforge.jnlp.Launcher.createApplication(Launcher.java:587)
	at net.sourceforge.jnlp.Launcher.launchApplication(Launcher.java:372)
	at net.sourceforge.jnlp.Launcher.access$200(Launcher.java:70)
	at net.sourceforge.jnlp.Launcher$TgThread.run(Launcher.java:654)
Caused by: net.sourceforge.jnlp.LaunchException: Fatal: Initialization Error: Unknown Main-Class. Could not determine the main class for this application.
	at net.sourceforge.jnlp.runtime.classloader.JNLPClassLoader.initializeResources(JNLPClassLoader.java:746)
	at net.sourceforge.jnlp.runtime.classloader.JNLPClassLoader.<init>(JNLPClassLoader.java:350)
	at net.sourceforge.jnlp.runtime.classloader.JNLPClassLoader.createInstance(JNLPClassLoader.java:423)
	at net.sourceforge.jnlp.runtime.classloader.JNLPClassLoader.getInstance(JNLPClassLoader.java:495)
	at net.sourceforge.jnlp.runtime.classloader.JNLPClassLoader.getInstance(JNLPClassLoader.java:468)
	at net.sourceforge.jnlp.Launcher.createApplication(Launcher.java:579)
	... 3 more
```

**Comment by sclassen** [comment]
Maybe I do not understand it all, but why do you expect a socket proxy to be used when all your resources are using https protocol??

**Comment by good4y0u** [comment]
> Maybe I do not understand it all, but why do you expect a socket proxy to be used when all your resources are using https protocol??

Why wouldn't I? The socket is opened to make the connection. This is a normal way to connect to and use a proxy 

>SOCKS, which stands for Socket Secure, is a network protocol that facilitates communication with servers through a firewall by >routing network traffic to the actual server on behalf of a client. SOCKS is designed to route any type of traffic generated by >any protocol or program

>Since SOCKS sits at layer 5, between SSL (layer 7) and TCP/UDP (layer 4), it can handle several request types, including HTTP, >HTTPS, POP3, SMTP and FTP. As a result, SOCKS can be used for email, web browsing, peer-to-peer sharing, file transfers and >more.

https://securityintelligence.com/posts/socks-proxy-primer-what-is-socks5-and-why-should-you-use-it/

**Comment by janakmulani** [comment]
@good4y0u Can you please provide screen shot from itw-settings  proxy settings and advanced settings? Also your deployment.properties file.  Thanks

**Comment by good4y0u** [comment]
**itw-settings** ; I want to note it doesn't work with 127.0.0.1 either
![itw-settings](https://user-images.githubusercontent.com/8586337/100420729-ea3a2f80-3054-11eb-824b-17ee9a0be948.PNG)

**deployment.properties** ; I want to note here that I did try a few different things, including locking the specific property in hopes it would force the connection. 

`#Netx deployment configuration
#Mon Nov 23 18:03:32 EST 2020
deployment.security.jsse.hostmismatch.warning=false
deployment.proxy.socks.port=8080
deployment.log=true
deployment.console.startup.mode=SHOW
ows.install4j.propertyUpdate=1606169430157
deployment.proxy.socks.host=localhost
deployment.proxy.type=1
deployment.proxy.locked=true`

**Comment by sclassen** [comment]
fixed with #369

**Comment by ddiolosa** [comment]
Is this issue back again?   I still see same issue with 1.8.0?
I tried HTTP Proxy and it works, but using SOCKS proxy is not.

No proxy found for 'http://192.168.122.25:80/ClientApplicationSplash.gif'. **Falling back to NO_PROXY**

Finished Proxy Selector
proxy bypass list: []
Socks proxy congfig: host 127.0.0.1 - port 1091
Ftp proxy congfig: host null - port -1
Https proxy congfig: host null - port -1
Http proxy congfig: host null - port -1
Manual proxy created
Selected ProxyProvider : MANUAL_HOSTS
Start Proxy Selector
